docs/content/docs/patterns/observability/_index.md

```diff
@@ -10,6 +10,6 @@ aliases: ["observability"]
 OSM's observability stack includes Prometheus for metrics collection, Grafana for metrics visualization, Jaeger for tracing and Fluent Bit for log forwarding to a user-defined endpoint. These are disabled by default but may be enabled during OSM installation with flags `--deploy-prometheus`, `--deploy-grafana`, `--deploy-jaeger` and `--enable-fluentbit`.
 
 ## Table of Contents
-- [Metrics - Prometheus and Grafana](./metrics.md)
+- [Metrics - Prometheus and Grafana](/docs/tasks_usage/metrics.md)
 - [Tracing - Jaeger](./tracing.md)
-- [Log forwarding - Fluent Bit](./logs.md)
\ No newline at end of file
+- [Log forwarding - Fluent Bit](/docs/tasks_usage/logs.md)
```

Comment:
These links lead to a 404, because the paths starting with `/` are relative to the root of the repo:
https://github.com/openservicemesh/osm/blob/main/docs/content/docs/patterns/observability/_index.md